DeusEx/Classes/DXRFixup.uc

```diff
@@ -139,6 +155,51 @@ function NYC1_FirstEntry()
     }
 }
 
+function Airfield_FirstEntry()
+{
+    local Mover m;
+    local Switch2 s;
+    local vector loc;
+    local rotator rotate;
+    
+    switch (dxr.localURL)
+    {
+        case "03_NYC_AirfieldHeliBase":
```

Comment:
wait, isn't dxr.localURL always all-caps? is switch/case case-sensitive in unrealscript?

Reply:
Huh, good catch.  Guess it must be, because this was working just fine for me...